acceptance/tests/rototillerTask/override_command_flag_with_env_var.rb

```diff
@@ -105,4 +105,38 @@ def remove_reserved_keys(h)
       end
     end
   end
+
+  step 'Test flags that are not required' do
+    override = (0...10).map { ('a'..'z').to_a[rand(26)] }.join
```

Comment:
we have {{random_string()}} in the acceptance lib
but it upcases.  that upcase should probably be removed.  one can upcase their own lazy self.
anyway, use that random_string helper here.